proc-start

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+#
+# Run single command from Procfile
+#
+# This script is a basic replacement for foreman when running on Docker. When
+# starting the docker instance, specify as command `script/start <type>`.
+# `type` is looked up in `Procfile` and the command is started. This allows e.g.
+# docker-compose to run multiple process/dyno types from a single image, without
+# needing to know the exact command (which may change over time).
+#
+if [ ! "$1" ]; then
+  echo "Usage: $0 <process type>" 1>&2
+  echo
+  echo "Note that some process types need the PORT environment variable."
+  exit 1
+fi
+
+CMD=`cat Procfile | grep "^$1:" | cut -d: -f2-`
+if [ ! "$CMD" ]; then
+  echo "Process type $1 not found in Procfile" 1>&2
+  exit 1
+fi
+
+exec /bin/sh -c "$CMD"
```

Comment:
i'm ok with the current `proc-start`, but i had the idea of replacing it with `rake`tasks in the meantime. e.g:
```ruby
task "cron" => :environment do
  system 'supercronic crontab'
end
```

Reply:
Nice idea for unification. It fails on one aspect, though: `proc-start` is an entry point that always works, regardless of the technology. Does one need to use `bundle exec` or not? Is the mail receiver actually a Ruby-program, or rather an MTA that runs a rake task on receival (which could be spawned by Ruby of course, but that is less efficient)? This is all agnostic to the sysadmin using the foodsoft image in a docker-compose configuration in the current situation.
Also, I have the intention to make Foodsoft work with things like Heroku and Dokku out of the box.